packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -165,6 +165,15 @@ class Sidebar extends React.Component<any, any> {
               to: '/shorthand-props',
             },
           },
+          {
```

Comment:
Now that we have the viewer we may even hide from public the example with all icons, and add link on the SvgIcon's docs page for the icons viewer... I am open for other ideas/suggestions here